requirements/edx/paver.txt

```diff
@@ -5,3 +5,4 @@ lazy==1.1
 path.py==7.2
 watchdog==0.7.1
 python-memcached
+libsass==0.10.0
```

Comment:
@ormsbee, is this the plain old version of LibSass (https://github.com/sass/libsass) or a python port (https://pypi.python.org/pypi/libsass). Seems like the latter, but I just wanted to check.